source/codegen/metadata/nirfsg/functions.py

```diff
@@ -0,0 +1,98 @@
+functions = {
+    "Init": {
+        "init_method": True,
+        "codegen_method": "public",
+        "cname": "niRFSG_init",
+        "parameters": [
+            {
+                "name": "resourceName",
+                "direction": "in",
+                "type": "ViRsrc"
+            },
+            {
+                "name": "idQuery",
+                "direction": "in",
+                "type": "ViBoolean"
+            },
+            {
+                "name": "reset",
+                "direction": "in",
+                "type": "ViBoolean"
+            },
+            {
+                "name": "vi",
+                "direction": "out",
+                "type": "ViSession",
+                "xxgrpc_type": "nidevice_grpc.Session"
```

Comment:
xx? I think this isn't needed.

Reply:
hah, yup, should have used `TODOTODO` or something :-)